concent_api/concent_api/settings/base.py

```diff
@@ -372,4 +372,4 @@
 # ETHEREUM_CHAIN =
 
 # Define dir where render_tools from golem repository should be stored
-BLENDER_RENDER_TOOLS_DIR = ''
+# BLENDER_RENDER_TOOLS_DIR = ''
```

Comment:
I think this should have a default in `base.py`. It should point at the directory where these files would if we were running in our Docker container.

Reply:
Done